eng/pipelines/branch-cleanup.yml

```diff
@@ -1,17 +1,44 @@
 parameters:
-- name: Repos
-  type: object
-  default:
-    - azure-sdk-for-android
-    - azure-sdk-for-c
-    - azure-sdk-for-cpp
-    - azure-sdk-for-go
-    - azure-sdk-for-ios
-    - azure-sdk-for-java
-    - azure-sdk-for-js
-    - azure-sdk-for-net
-    - azure-sdk-for-python
-
+Repos:
+  - RepoOwner: Azure
+    RepoName: azure-sdk-for-android
+    BranchPrefix: "sync-eng/common-"
+  - RepoOwner: Azure
+    RepoName: azure-sdk-for-c
+    BranchPrefix: "sync-eng/common-"
+  - RepoOwner: Azure
+    RepoName: azure-sdk-for-cpp
+    BranchPrefix: "sync-eng/common-"
+  - RepoOwner: Azure
+    RepoName: azure-sdk-for-go
+    BranchPrefix: "sync-eng/common-"
+  - RepoOwner: Azure
+    RepoName: azure-sdk-for-ios
+    BranchPrefix: "sync-eng/common-"
+  - RepoOwner: Azure
+    RepoName: azure-sdk-for-java  
+    BranchPrefix: "sync-eng/common-"
+  - RepoOwner: Azure
+    RepoName: azure-sdk-for-js
+    BranchPrefix: "sync-eng/common-"
+  - RepoOwner: Azure
+    RepoName: azure-sdk-for-net
+    BranchPrefix: "sync-eng/common-"
+  - RepoOwner: Azure
+    RepoName: azure-sdk-for-python
+    BranchPrefix: "sync-eng/common-"
+  - RepoOwner: Azure
+    RepoName: azure-docs-sdk-java
+    BranchPrefix: ("daily/" + (Get-Date (Get-Date).AddDays(-7) -UFormat "%Y-%m-%d"))
```

Comment:
I assume this branch pattern is still a work in progress because as it stands I'm not sure it will work.